Engine/script/cc_instance.cpp

```diff
@@ -80,38 +80,38 @@ struct ScriptCommandInfo
 const ScriptCommandInfo sccmd_info[CC_NUM_SCCMDS] =
 {
     ScriptCommandInfo( 0                    , "NULL"              , 0, kScOpNoArgIsReg ),
-    ScriptCommandInfo( SCMD_ADD             , "add"               , 2, kScOpOneArgIsReg ),
-    ScriptCommandInfo( SCMD_SUB             , "sub"               , 2, kScOpOneArgIsReg ),
+    ScriptCommandInfo( SCMD_ADD             , "addi"              , 2, kScOpOneArgIsReg ),
+    ScriptCommandInfo( SCMD_SUB             , "subi"              , 2, kScOpOneArgIsReg ),
     ScriptCommandInfo( SCMD_REGTOREG        , "mov"               , 2, kScOpTwoArgsAreReg ),
-    ScriptCommandInfo( SCMD_WRITELIT        , "memwritelit"       , 2, kScOpNoArgIsReg ),
+    ScriptCommandInfo( SCMD_WRITELIT        , "memcpy"            , 2, kScOpNoArgIsReg ),
```

Comment:
I'm not sure this instruction was intended as a memcpy. It really is used for writing a literal value to memory and will fail if outside the size values of (1,2,4).

The original implementation did use a memcpy but it will read out of bounds memory if > 4 (and I think it's a little buggy as it doesn't take into account big endian systems like the other memwrite instructions do).

It looks like the compiler currently only uses this instruction for clearing pointers, so it would be interesting to see if earlier versions used other literal values.